### PR TITLE
DSERV-55-gene-name

### PR DESCRIPTION
**Pull request by mingjiecn**
1. index gene name that has no Entrez GeneID
2. update function get_matrix_file_download_url from dataset metadata
3. skip the line for footprint file if the seq contains N

### CODE REVIEW

File: tests/conftest.py
Comment (ottojolanki) [inline tests/conftest.py]: We should decide on single or double quotes (I have no opinion which) and stick to it. Just do not mix.

**Review by ottojolanki — CHANGES_REQUESTED** [review]
We need to either use double quotes or single quotes but not both. Other than that LGTM, no need to review again after fixing the quotes.

**Comment by mingjiecn** [comment]
merged DSERV-55-gene-name2 instead. https://github.com/ENCODE-DCC/genomic-data-service/pull/60

**Pull request closed without merging**
